HO adding all article files. (#149)
hassan-obeid/tr_b_causal_2020: article/sections/_1_intro_generic.tex

\section{Introduction}
\label{sec:intro}

In transportation, we often build models that are used for policy evaluation.
Specifically, we often use behavioral models to evaluate the impact of external interventions on travel outcomes. 
These evaluations are excplicitly causal: we are interested in how the 
system reacts to \textit{interventions}. 

> Yet, when these models are 
> developed, causality is often not considered. Furthermore, when 
> causal concepts are accounted for, the process is done implicitly without a 
> formal framework. 

Reviewer: Here, it seems like there is a disconnect from the previous sentences. You state "we" in the previous sentences, but create some distance here. Maybe the following?
```suggestion
Yet, when we develop these models, we often look causality.
Even in situations where we account for causality, we do so implicitly without using a formal framework.
```

Reviewer 2: "Yet when we develop these models, we often **overlook** causality"

This chapter is concerned with filling this gap. 
Specifically, we focus on addressing the existing disconnect between the fields of travel demand modeling and causal inference. 
The chapter is motivated by the current lack of use of methods and findings from 

> the causal inference literature in travel demand modeling. 

Reviewer: I feel these three sentences are saying pretty much the same thing. I am yet to find a good replacement that could combine the three of them.

While the field of transportation demand modeling could benefit greatly from 
incorporating causal inference techniques, there are barriers that have slowed this integration. 
These barriers stem from the difference between the types of problems transportation demand modelers deal with and those that are typically studied in the causal inference literature. 

> Perhaps the fundemental difference is that demand modelers are typically trying to forecast the impacts of policies that haven't been implemented or seen before.

Reviewer: I would add a contrast example from the side of causal inference.
I also would not say that causal inference doesn't try to do that. Maybe observational causal inference does but not all causal inference.

```suggestion
An example of this difference is that travel demand modelers are typically trying to forecast the impacts of policies that have not been implemented yet.
```

Reviewer 2: agreed, the people from structural economics would probably fall over in horror if they felt disregarded (imagine Nobel Laureate Jim Heckman who explicitly writes about the need to forecast policy impacts for unseen policies).

> Forecasting the effects of unseen interventions requires additional work and a change to the typical causal modeling workflow.
> In particular, we must translate a given policy (treatment) into a set of characteristics and variables that exist in the data and system at hand.
> (For a more thorough discussion of this and other barriers, please refer to \citet{brathwaite_2018_causal}.) 

Reviewer: ```suggestion
Forecasting the effects of yet to be implemented interventions requires additional work and a change to the typical causal modeling workflow.
In particular, we must break the intervention of interest into a set of characteristics and variables that exist in the data and system at hand.
(For a more thorough discussion of this and other barriers, please refer to \citet{brathwaite_2018_causal}.) 
```

> While these barriers complicate the efforts of demand modelers, there is still a lot to gain from
> incorporating causal inference techniques where appropriate and from contributing
> to the causal inference literature where it's lacking.

Reviewer: ```suggestion
While these barriers complicate the efforts of travel demand modelers, there is still a lot to gain from 1)
incorporating causal inference techniques where appropriate and 2) contributing
to the causal inference literature where needed.
```

> The relevance of this topic now is motivated by the significant boost in the causal inference literature recently, both in the potential outcomes and the causal graphical modeling frameworks.
> The goal of this chapter is to formalize a workflow for approaching 
> transportation demand modeling problems from a causal perspective. 

Reviewer: My suggestion is to stick with either "travel demand modeling" or "transportation demand modeling" throughout the document.

```suggestion
The current relevance of this topic is motivated by the recent significant boost in the causal inference literature, both in the potential outcomes and the causal graphical modeling frameworks.
The goal of this chapter is to formalize a workflow for approaching 
travel demand modeling problems from a causal perspective. 
```

Reviewer 2: my suggestion would be to go even broader and speak of choice modelling (using transportation as an example). Choice modelling is already a small enough niche in the world, no need to hem ourselves in even further.

We will draw heavily on the use of directed acyclic graphs (DAGs) formalized by \citet{pearl_causality_2000} as a means of representing the modeler's knowledge and assumptions about a given problem. 
The chapter will provide an overview of DAGs, the 
testable implications that come with one's causal representation, the main 
tests that one could do to falsify or justify a given causal graph, and then how 

> to use a causal graph to estimate the causal relationships of interest. 

Reviewer: Is this sentence supposed to summarize how the chapter is ordered? If so, I think it should be edited.

We will demonstrate the use of this framework through simulations, where we 
clearly show the benefits and implications of this approach as opposed to 

> traditional approaches. 

Reviewer: I'm not sure why when I read this sentence, "implications" sounded like "drawbacks" to me. Is it intended to be that way?

> The last part of this chapter deals with the more complicated issue of latent 
> confounding, where one variable confounds two or more variables in the causal 
> graph. This type of confounding creates variations in the outcome variable that are not caused by 
> the confounded variables but are correlated with it, which biases the estimated 
> causal effects of those variables if nothing is done to account for the 
> confounding. 

Reviewer: ```suggestion
The last section of this chapter addresses the more complicated issue of latent 
confounding; where one variable confounds two or more variables in a causal 
graph. This type of confounding creates variations in the outcome variable that are not directly caused by 
the confounded variables but are correlated with it, biasing the estimated 
causal effects of those variables if nothing is done to account for such
confounding. 
```
